variants/heltec_wifi_kit_32/pins_arduino.h

```diff
@@ -52,8 +55,8 @@ static const uint8_t T4 = 13;
 static const uint8_t T5 = 12;
 static const uint8_t T6 = 14;
 static const uint8_t T7 = 27;
-static const uint8_t T8 = 33;
-static const uint8_t T9 = 32;

```

Comment:
Actually this is correct ;) 32 is T9 and 33 is T8

Reply:
Well that's interesting.  The pinout diagram states gpio 33 = t8, but the espressif PDF states gpio 32 = t8.